ifrs17/Import/ImportStorage.ipynb

```diff
@@ -341,11 +341,11 @@
                 "\n    public IEnumerable<IfrsVariable> GetIfrsVariables(string dataNode) => IfrsVariablesByImportIdentity.TryGetValue(dataNode, out var variableCollection) ? variableCollection : Enumerable.Empty<IfrsVariable>();",
                 "\n    ",
                 "\n    public double[] GetValues(ImportIdentity id, Func<RawVariable, bool> whereClause) => GetRawVariables(id.DataNode).Where(v => (v.AocType, v.Novelty) == id.AocStep && whereClause(v)).Aggregate()?.Values ?? Enumerable.Empty<double>().ToArray();",
-                "\n    public double GetValue(ImportIdentity id, Func<IfrsVariable, bool> whereClause) => GetIfrsVariables(id.DataNode).Where(v => (v.AocType, v.Novelty) == id.AocStep && whereClause(v)).Aggregate()?.Value ?? 0;",
+                "\n    public double GetValue(ImportIdentity id, Func<IfrsVariable, bool> whereClause, int projection = 0) => GetIfrsVariables(id.DataNode).Where(v => (v.AocType, v.Novelty) == id.AocStep && whereClause(v)).Aggregate()?.Values.ElementAtOrDefault(projection) ?? 0;",
```

Comment:
the .Aggregate() here won't work. As the variable has double[] prop.
We can .Select(x => x.Values).AggregateDoubleArray()